runtime/gc_vlhgc/AllocationContextBalanced.hpp

```diff
@@ -58,7 +58,7 @@ class MM_AllocationContextBalanced : public MM_AllocationContextTarok
 	MM_RegionListTarok _discardRegionList; /**< The list of MPBP regions currently privately owned by this context but either too full or too fragmented to be used to satisfy allocations.  These regions must be flushed back to the subspace before a collection */
 	MM_RegionListTarok _flushedRegions; /**< The list of MPBP regions which have been flushed from active use, for a GC, and have unknown stats (at any point after the GC, however, these regions could all be migrated to _ownedRegions) */
 	MM_RegionListTarok _freeRegions; /**< The list regions which are owned by this context but currently marked as FREE */
-	MM_RegionListTarok _idleMPBPRegions; /**< The list regions which are owned by this context, currently marked as BUMP_ALLOCATED_IDLE, but contain no objects (this also implies that they can migrate to other contexts on this node, for free, since the receiver isn't using them) */
+	MM_RegionListTarok _idleMPBPRegions; /**< The list regions which are owned by this context, currently marked as ADDRESS_ORDERED_IDLE, but contain no objects (this also implies that they can migrate to other contexts on this node, for free, since the receiver isn't using them) */
```

Comment:
_idleMPBPRegions -> _idleRegions (BP stands for BumpPointer that is going away)

Reply:
there are a few more spots in the code and comments that use BP or MPBP acronym that you need to rename